tests/languages/markdown/url_feature.test

```diff
@@ -108,8 +108,8 @@ bar](http://prismjs.com)
 			"foo ",
 			["bold", [
 				["punctuation", "__"],
-				["content", ["bar"]],
-				["punctuation", "__"]
+				["content", ["bar_"]],
+				"_"

```

Comment:
Why did this change? This has nothing to do with your PR? Did you locally change any other `component/*.js` files?

If not, what is your `node -v` and `npm -v`?